helm-charts/ironic/templates/deployment.yaml

```diff
@@ -64,7 +64,7 @@ spec:
           envFrom:
           {{- range .envFrom }}
             - configMapRef:
-                name: {{ $relname }}-{{ .configMapRefname }}
```

Comment:
Not sure if you really need this change.

Reply:
Since I am using static configmapName. I no longer need to append that with release name